docs/api.md

```diff
@@ -853,6 +853,37 @@ client.Bridge.update('brg-65dhjbanasiei', {
 		}
 	});
 ```
+**Example**  
+```js
+// end bridge
+var bridgeOptions = {
+	callIds: []
+};
+
+client.Bridge.update("{bridgeId}", bridgeOptions)
+.then(function () {
+	// continue
+});
+```
+**Example**  
+```js
+// Add two calls to bridge the remove one
+var bridgeOptions = {
+	bridgeAudio : true,
+	callIds: ["{callId1","callId2"]
+};
+
+client.Bridge.update("{bridgeId}", bridgeOptions)
+.then(function () {
+	var removeCall2 = {
```

Comment:
Data objects with verb names irk me. This whole block would be simpler with this inlined...

```javascript
.then(client.Bridge.update("{bridgeId}", {
  callIds: ["{callId1}"]
}))
```